src/reference/v8.json

```
@@ -364,6 +364,24 @@
         }
       }
     },
+    "encoding": {
+      "type": "enum",
+      "values": {
+        "mapbox": {
+          "doc": "Mapbox Vector Tiles. See http://github.com/mapbox/vector-tile-spec for more info."
+        },
```

To my knowledge, MVT initially stood for Map Vector Tile

```suggestion
        "mvt": {
          "doc": "Map Vector Tiles. See http://github.com/mapbox/vector-tile-spec for more info."
        },
```